### ISSUE TITLE
Minimize hello_world example

### ISSUE THREAD

**Issue by zjuturtle** [google/mediapipe#297]
I want to use mediapipe in a very limited space device. I found that after use 

```
# bazel build -c opt --define MEDIAPIPE_DISABLE_GPU=1   mediapipe/examples/desktop/hello_world:hello_world
# strip -s hello_world
```

I still got about 2.3M size. It is too big for a hello_world. Any ideas I can shrink the binary output size?

**Comment by jiuqiant** [comment]
FYI, if you build the framework, the static library size is 561 k.

```
$ bazel build -c opt --define MEDIAPIPE_DISABLE_GPU=1 -s  //mediapipe/framework:calculator_graph

...
Target //mediapipe/framework:calculator_graph up-to-date:
  bazel-bin/mediapipe/framework/libcalculator_graph.a
  bazel-bin/mediapipe/framework/libcalculator_graph.pic.a
  bazel-bin/mediapipe/framework/libcalculator_graph.so

$ ls -lh bazel-bin/mediapipe/framework/libcalculator_graph.a
-r-xr-xr-x. 1 jqtang jqtang 561K Dec  5 19:20 bazel-bin/mediapipe/framework/libcalculator_graph
.a
```

**Comment by zjuturtle** [comment]
I've tired to use libcalculator_graph.a as a framework. I found that when I try link to libcalculator_graph.a, I got a lot undefined symbols(from absl to mediapipe). Then I use nm to check what exactly in the libcalculator_graph.a. 
For example, I believe libcalculator_graph.a should have mediapipe::CalculatorContext::XXX symbol.

```
nm  -C libcalculator_graph.a | grep 'mediapipe::CalculatorContext:'
[None displayed]
```

But it did have mediapipe::CalculatorGraph::XXX symbol.

```
nm -C libcalculator_graph.a | grep 'mediapipe::CalculatorGraph::'
[Displayed a lot symbols]
```

Maybe libcalculator_graph.a is not included everything needed in the framework?
